Adafruit_L3GD20_U.cpp

```diff
@@ -64,6 +64,7 @@ byte Adafruit_L3GD20_Unified::read8(byte reg)
   #endif
   Wire.endTransmission();
   Wire.requestFrom((byte)L3GD20_ADDRESS, (byte)1);
+  while (!Wire.available()); // Wait for data to arrive.
```

Comment:
There is no need to wait for data to arrive.
After the Wire.requestFrom() the I2C transmission has completely finished and the data is in the buffer.